content/intro-to-storybook/ember/en/get-started.md

```diff
@@ -0,0 +1,106 @@
+---
+title: 'Storybook for Ember tutorial'
+tocTitle: 'Get started'
+description: 'Setup Storybook in your development environment'
+---
+
+Storybook runs alongside your app in development mode. It helps you build UI components isolated from the business logic and context of your app. This edition of Learn Storybook is for Ember; other editions exist for [React](/react/en/get-started), [React Native](/react-native/en/get-started), [Vue](/vue/en/get-started), [Angular](/angular/en/get-started) and [Svelte](/svelte/en/get-started).
+
+![Storybook and your app](/intro-to-storybook/storybook-relationship.jpg)
+
+## Setup Ember Storybook
+
+We’ll need to follow a few steps to get the build process set up in your environment. To start with, we want to use [ember-cli](https://github.com/ember-cli/ember-cli) to setup our build system, and enable [Storybook](https://storybook.js.org/) and [Qunit](https://qunitjs.com/) testing in our app. Let’s run the following commands:
+
+```bash
+# Create our application:
+ember new taskbox
+
+cd taskbox
+
+# Add Storybook:
+npx -p @storybook/cli sb init
+
+# Add Ember storybook adapter
+ember install @storybook/ember-cli-storybook
+```
+
+<div class="aside">
+<p>
+At the time of writing this tutorial adding the @storybook/ember-cli-storybook, will yield the following message :
+</p>
+The <code>`ember generate <entity-name>`</code> command requires an entity name to be specified. <br/>For more details, use `ember help`.
+<p>Don't worry about it. Everything is installed and properly configured for you.</p>
+</div>
+
+We'll need to install some additional packages in our app, more specifically one Ember addon and one package, both this one will help us out shortly. Run the following commands:
+
+```bash
+ember install ember-truth-helpers
+```
+
+Followed by:
+
+```
+npm install -D npm-run-all
+```
+
+And finally make a small change to our <code>package.json</code> to allow Storybook to work properly with our Ember app.
+
+Add the following entry to your scripts:
+
+```javascript
+// package.json
+{
+    "scripts":{
+        "storybook-dev":"npm-run-all --aggregate-output --continue-on-error --parallel start storybook",
+        "prebuild-storybook":"ember build"
```

Comment:
Maybe include the existing `start`/`storybook`/`build-storybook` scripts here otherwise people might remove them.

Reply:
@tmeasday off course. I will update it shortly.

Reply:
don't forget the `,` on line 59 (and thank you for this it's been supremely helpful!)